modules/example-apache-httpd-configuration.adoc

```diff
@@ -7,7 +7,21 @@
 
 The basic identify provider (IDP) configuration in {product-title} 4 requires
 that the IDP server respond with JSON for success and failures. You can use CGI
-scripting in Apache HTTPD to accomplish this. This section provides examples.
+scripting in Apache HTTPD to accomplish this. This section provides examples
+with an httpd server configured as running a pod in cluster.
```

Comment:
Current PR does not include how to configure it as `running a pod in cluster`, how to configure it as a pod in cluster is shown in bug comment 11's steps.

Reply:
If the PR would not include how to configure it as a pod in cluster, then readers don't know how below "basicauth-xxia1" user credential is prepared.